flax/linen/module.py

```diff
@@ -742,9 +834,21 @@ def _call_wrapped_method(self, fun, args, kwargs):
           self.sow('intermediates', fun_name, y)
       if add_call_info:
         _args, _kwargs, _y = jax.tree_util.tree_map(
-          _get_value_representation, (args, kwargs, y), is_leaf=lambda x: x is None)
+            _get_value_representation,
+            (args, kwargs, y),
+            is_leaf=lambda x: x is None,
+        )
         _context.call_info_stack[-1].calls.append(
-          _CallInfo(call_index, scope_path, type(self), fun.__name__, _args, _kwargs, _y))
+            _CallInfo(
```

Comment:
Many of these lines become much less dense. I guess it goes a bit against how JAX usually formats their code, which is more dense.

Reply:
black (and thus pyink) are mostly non-configurable